src/OpenSage.Game/Client/Drawable.cs

```diff
@@ -369,6 +369,11 @@ public void ShowSubObjectPermanently(string subObject)
             _hiddenSubObjects.Remove(subObject);
         }
 
+        public new void Dispose()
+        {
+            Destroy();
+        }
+
```

Comment:
Instead of doing this, should we maybe add all drawables to the `DisposableBase` via `AddDisposable`, so we can leverage what we're already inheriting?